single-line-diagram/single-line-diagram-core/src/main/resources/animations.css

```diff
@@ -31,3 +31,57 @@
 .sld-arrow-animation-low-speed { --sld-arrow-animation-parameters: 3s infinite linear }
 .sld-arrow-animation-average-speed { --sld-arrow-animation-parameters: 2s infinite linear }
 .sld-arrow-animation-high-speed { --sld-arrow-animation-parameters: 1s infinite linear }
+
+:root {
+  --sld-overload-color: #ffeb3b; /* Yellow for overload */
+  --sld-overvoltage-color: #5a0101; /* Red for overvoltage */
+  --sld-undervoltage-color: #0000ff; /* Blue for undervoltage */
+}
```

Comment:
They're constant and they're used only once each, so you can use directly the value inside the animations.

Reply:
done

Reply:
agree , i changed it